swap_meet/vendor.py

```diff
@@ -1,2 +1,75 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if not inventory:
+            inventory = []
+        self.inventory = inventory
+
+    def add(self, item_to_add):
+        self.inventory.append(item_to_add)
+        return item_to_add
+    
+    def remove(self, item_to_remove):
+        # check that item is in inventory
+        if item_to_remove not in self.inventory:
+            return False
+
+        self.inventory.remove(item_to_remove)
+        return item_to_remove
+
+    def get_by_category(self, category):
+        items_in_category = []
+        
+        # check each item in inventory and add items of given category to list
+        for item in self.inventory:
+            if item.category == category:
+                items_in_category.append(item)
+        
+        return items_in_category
+    
+    def swap_items(self, friend, my_item, their_item):
+        # check that each vendor has item to swap
+        if not my_item in self.inventory or not their_item in friend.inventory:
+            return False
+        
+        # swap items in each vendor's inventory
+        self.add(their_item)
+        friend.inventory.remove(their_item)
```

Comment:
We could use the `remove` function we wrote on the `friend` instance of the Vendor class:
```suggestion
        friend.remove(their_item)
```